README.md

```diff
@@ -382,8 +382,8 @@ this feature in your project's build configuration.
 
 The mutations that Dredd applies rely on a number of function definitions that must be present at a suitable point in each file that Dredd mutates.
 These are referred to as the Dredd *prelude*.
-Dredd uses a heuristic to determine where to place its prelude.
-Sometimes this heuristic may not find an appropriate place, e.g. because of limitations or bugs in Clang's libTooling (see [this issue](https://github.com/mc-imperial/dredd/issues/322) for example).
+Dredd inserts prelude at the beginning of source files that may produce mutants.
```

Comment:
"inserts prelude" -> "inserts its prelude"

"at the beginning of source files that may produce mutants" -> "at the beginning of each mutated source file"

Reply:
done